docs/tutorial/04-capabilities.md

```diff
@@ -42,7 +42,7 @@ language's design.
 
 :::
 
-This tutorial builds on the [previous `Resource` tutorial](./03-resources.md).
+This tutorial builds on the [previous `Resource` tutorial](./03-resources.mdx).
```

Comment:
Here and below: The file is named [03-resources.md](https://github.com/onflow/cadence-lang.org/blob/brian-doyle/content-updates-1/versioned_docs/version-0.42/tutorial/03-resources.md), so this should not be changed. Same below